### PR TITLE
Serial part 1.0.0 renaming

### LINKED ISSUE #79: [New Model]: [Traceability] New model SerialPartAsBuilt (replacement of SerialPartTypization)
## Model Description
We want to create a new model named SerialPartAsBuilt which will replace the existing model SerialPartTypization. The reason for the replacement is to match our naming convention from BomAsPlanned and other BoMs. 

## Relevant Standards
New model will replace existing SerialPartTypization therefore not appicable

## Example Data
New model will replace existing SerialPartTypization, therefore not appicable

## MS1 Criteria

- [x] The proposed aspect model does not exist already
- [x] The proposed aspect model does not extend an existing aspect but introduces completely new functionality
- [x] The proposal references a Catena-X use case
- [x] Relevant standards are mentioned/linked
- [x] A first draft of the data structure is provided in an example

### PR DESCRIPTION
**Pull request by jacewski-bosch**
## Description

We want to create a new model named SerialPartAsBuilt which will replace the existing model SerialPartTypization. The reason for the replacement is to match our naming convention from BomAsPlanned and other BoMs.

Closes #79 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the BAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar bamm-cli.jar -i \<path-to-aspect-model\> -v ). The  BAMM CLI is available [here](https://openmanufacturingplatform.github.io/sds-documentation/sds-developer-guide/dev-snapshot/tooling-guide/bamm-cli.html) and in [GitHub](https://github.com/OpenManufacturingPlatform/sds-sdk/releases)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "name" and "description"** in English language. 
- [x] **no duplicate names or preferredNames** within an Aspect (e.g. a Property and the referenced Characteristic should not have the same name)
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the BAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'

### CODE REVIEW

File: io.catenax.serial_part/1.0.0/SerialPart.ttl [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+:PartNameCharacteristic a samm:Characteristic ;
+   samm:preferredName "Part Name Characteristic"@en ;
+   samm:description "Part Name in string format from the respective system in the value chain"@en ;
+   samm:dataType xsd:string .
+
+:ClassificationCharacteristic a samm-c:Enumeration ;
+   samm:preferredName "Classification Characteristic"@en ;
+   samm:description "A part type must be placed into one of the following classes: 'component', 'product', 'software', 'assembly', 'tool', or 'raw material'."@en ;
+   samm:see <http://private.pdm-if.org/web/pdm-if/recommended-practices1> ;
```
Comment (bs-jokri) [inline io.catenax.serial_part/1.0.0/SerialPart.ttl:193]: Url don't seem to be accessible

**Review by bs-jokri — CHANGES_REQUESTED** [review]
please add release notes and check comment

**Review by bs-jokri — CHANGES_REQUESTED** [review]
please add release notes and check comment

File: io.catenax.serial_part/1.0.0/metadata.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{ "status" : "release"}
```
[the remark this answers is not in the document]
Comment (jacewski-bosch) [reply io.catenax.serial_part/1.0.0/metadata.json:1]: done.

File: io.catenax.serial_part/1.0.0/SerialPart.ttl (hunk 1 above)
Comment (jacewski-bosch) [reply io.catenax.serial_part/1.0.0/SerialPart.ttl]: fixed.

**Comment by jacewski-bosch** [comment]
modeling_team

**Review by bs-jokri — APPROVED** [review]
LGTM

**Comment by BirgitBoss** [comment]
I think the Characteristic of ManufacutringCharateristic should be "SingleEntity"
I think the Characteristic of PartTypeInformationCharacteristic should be "SingleEntity"
For the string Characteristics you should consider to use the predefined characteristics "Text" or the characteristic "Code"

**Review by bs-jokri — APPROVED** [review]
LGTM

**Pull request merged**